summit-pitch.md

```diff
@@ -0,0 +1,43 @@
+# Summit pitch
+
+Questions: who shall be the person that pitches?
+
+## Title
+*The title of your talk/discussion topic. This will appear in the Language Summit schedule on the conference website.*
+
+Fearless Concurrency in Python
+
+
+## Key discussion item
+
+> *What is the "thing" you want to discuss in front of mostly Python core developers? What decision to be made? What are your questions, and proposed solutions? This should fit in 10 minutes, to allow for 20 minutes of discussion. Therefore this should be very focused. This field will only be read by Language Summit co-chairs to determine the talk acceptance. You can include a rough outline, useful links, sample discussions, etc.*
+
+
+We want to discuss a dynamically checked ownership model for Python and a "fearless" concurrency model that it enables.
+The ownership model is based on "regions" and "deep immutability" and allows existing Python data structures to be used in multi-threaded programs.
+
+The presentation will be based on collaboration with the Faster CPython team at Microsoft and
+prototyping work done on a fork of the CPython interpreter.
+The initial design is documented in a research paper that is conditionally accepted to appear at the ACM PLDI'25 conference.
+[Note that the paper is not yet public, but will be available by the time of the Language Summit.]
+
+The aim of the presentation is to get feedback on the ideas, and to discuss the implications of the model for Python.
+Getting something like this into Python would be a big change, and we want to discuss this with the core developers.
+We will also discuss how the model can be implemented in CPython, and what changes would be needed to the language and runtime to support it.
+
+
+## The public pitch
+*This will be shown in the Language Summit schedule on the conference website and published ahead of the event. Write one or two short paragraphs to pitch this topic to the attendees. This is your chance to encourage people to discuss the topic with you!*
+
+With the move to free-threaded Python, the challenges of concurrency are coming to Python developers.
```

Comment:
It could be made clearer that your concurrency model is not tied to the free threading issue. You mention subinterpreters below, but you open with free-threading and mention it later more than subinterpreters, so it dominates and I think this may create an impression that this is the motivation.

Does it makes sense to speak about concurrency in this paragraph, rather than free threading, and then mention free threading and subinterpreters, once each?

Reply:
We didn’t want to be seen as a competing proposal, especially given that PEP703 is already accepted. I’ll craft something! Thanks for the suggestion.